core/templates/pages/admin-page/roles-tab/admin-roles-tab.component.ts

```diff
@@ -124,12 +131,38 @@ export class AdminRolesTabComponent implements OnInit {
     });
   }
 
+  openTranslationCoordinatorrRoleEditor(): void {
```

Comment:
Again, please fix the typo. It should be `openTranslationCoordinatorRoleEditor`.